README.md

```diff
@@ -341,16 +341,21 @@ These are the default commands and bindings available for file/scroll/window/tab
 
 #### Explorer/list navigation
 
-| Key                                | VSCode Command                  |
-| ---------------------------------- | ------------------------------- |
-| <kbd>j</kbd> / <kbd>k</kbd>        | `list.focusDown/Up`             |
-| <kbd>h</kbd> / <kbd>l</kbd>        | `list.collapse/select`          |
-| <kbd>Enter</kbd>                   | `list.select`                   |
-| <kbd>gg</kbd>                      | `list.focusFirst`               |
-| <kbd>G</kbd>                       | `list.focusLast`                |
-| <kbd>o</kbd>                       | `list.toggleExpand`             |
-| <kbd>C-u</kbd> / <kbd>C-d</kbd>    | `list.focusPageUp/Down`         |
-| <kbd> / </kbd> / <kbd>Escape</kbd> | `list.toggleKeyboardNavigation` |
+| Key                                                   | VSCode Command                  |
+| ----------------------------------------------------- | ------------------------------- |
+| <kbd>j</kbd> / <kbd>k</kbd>                           | `list.focusDown/Up`             |
+| <kbd>h</kbd> / <kbd>l</kbd>                           | `list.collapse/select`          |
+| <kbd>Enter</kbd>                                      | `list.select`                   |
+| <kbd>gg</kbd>                                         | `list.focusFirst`               |
+| <kbd>G</kbd>                                          | `list.focusLast`                |
+| <kbd>o</kbd>                                          | `list.toggleExpand`             |
+| <kbd>C-u</kbd> / <kbd>C-d</kbd>                       | `list.focusPageUp/Down`         |
+| <kbd>z</kbd> <kbd>o</kbd> / <kbd>z</kbd> <kbd>O</kbd> | `list.expand`                   |
+| <kbd>z</kbd> <kbd>c</kbd>                             | `list.collapse`                 |
+| <kbd>z</kbd> <kbd>C</kbd>                             | `list.collapseAllToFocus`       |
+| <kbd>z</kbd> <kbd>a</kbd> / <kbd>z</kbd> <kbd>A</kbd> | `list.toggleExpand`             |
```

Comment:
To help with discovery, 1-2 line mention the readme is worth it. But the high-fidelity listing of every mapping is not worth it. Users should instead check vscode shortcuts UI to see exact mappings. The readme should just give a useful hint like:

> In the vscode File Explorer some familiar vim commands are configured by default:
> - for navigation: `j`, `k`, `gg`, `G`
> - for expanding/collapsing directories: `zo`, `zc`, etc.
> - run the `Preferences: Open Keyboard Shortcuts` vscode command to see the complete list of keybindings.

This is a more maintainable pattern that we can follow in all sections of the readme, which will result in a less noisy, more maintainable, and still useful overview.

Reply:
@justinmk Would you mind doing this? I don't know which place would be the best to insert these. I will revert my `README.md` changes accordantly later.

Reply:
Ok, not a blocker

Reply:
I do agree, but
1. until this is cleaned up, I'd rather include the list in the readme. I'll revert myself
2. unfortunately discovery in keybindings editor is difficult because there is no way to filter by extension